src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -177,14 +183,14 @@ struct program_node {
     // do not modify primitive directly to keep synchronisation with graph
     std::shared_ptr<const primitive> get_primitive() const { return desc; }
     // primitive modification functions
-    void set_output_padding(padding const& padd) {
+    void set_output_padding(padding const& padd, size_t idx = 0) {
```

Comment:
Can we do for all outputs at once? 

e.g., 
void set_output_paddings(padding const& padd) {
     for (auto o : output_layouts) {
           o.data_padding = padd;
     }
}

Reply:
Currently, this API is still needed because new updated layouts in `set_output_layouts` are used by accessing `merge_output_padding` -> `set_output_padding` with output index.